File: java/LCCI_01_01.java
class Solution {
    public boolean isUnique(String astr) {
        int[] record = new int[26];
        Arrays.fill(record,0);
        for(int i  = 0 ; i < astr.length() ; i++){
            if (record[astr.charAt(i) - 'a'] != 0){
                return false;
            }
            record[astr.charAt(i) - 'a'] += 1;
        }
        return true;
    }
}

File: python/LCCI_01_01.py
class Solution:
    def isUnique(self, astr: str) -> bool:
        mark = 0
        for char in astr:
            move_bits = ord(char) - ord('a')
            if(mark & (1 << move_bits) != 0):
                return False
            else:
                mark |= (1 << move_bits)
        return True